.mailmap

```diff
@@ -91,3 +91,7 @@ Ignacio Tomas <itomas@sandia.gov>
 
 #
 Nicole Hayes <hayesni4@msu.edu>
+
+#
+Fabian Castelli <fabian.castelli@kit.edu>
+Fabian Castelli <fabian.castelli@kit.edu> <50630942+gfcas@users.noreply.github.com>
```

Comment:
Do we still need to do this manully?

Reply:
@tjhei maybe you know about that? Personally I have no knowledge on these internals.

Reply:
There shouldn't really be any need for this if you have set up your GitHub account properly (and we don't already have commits with the "wrong" e-mail-address in the repository). I guess this was just a quick and dirty solution.

Reply:
So nothing to do for me?

Reply:
Just remove these lines.

Reply:
OK